udata/core/reuse/models.py

```diff
@@ -40,12 +41,14 @@ def hidden(self):
                     db.Q(datasets__0__exists=False) |
                     db.Q(deleted__ne=None))
 
-
+decorator = mdstrip_field('description')
+@decorator
```

Comment:
I'm able to use the decorator as expected (`@mdstrip_field('description')`) with that code:

```python
# -*- coding: utf-8 -*-
from mongoengine.signals import pre_save_post_validation

from udata.frontend.markdown import mdstrip


def mdstrip_field(field_name):

    def apply_mdstrip(sender, document, **kwargs):
        if (field_name in document._get_changed_fields() or
                getattr(document, '_created')):
            initial_value = getattr(document, field_name)
            stripped_value = mdstrip(initial_value)
            document.__setattr__(field_name + '_rendered', stripped_value)

    class Inner(object):

        def __init__(self, cls):
            self.cls = cls

        def __getattr__(self, attr):
            return getattr(self.cls, attr)

        def __call__(self, *args, **kwargs):
            pre_save_post_validation.connect(apply_mdstrip, sender=self.cls)
            return self.cls(*args, **kwargs)

    return Inner
```

BUT, it still requires some work to make all tests pass (only `udata/tests/test_model.py` are OK now). It might be a good starting point.